lib/iris/config.py

```diff
@@ -148,13 +134,6 @@ def get_dir_option(section, option, default=None):
 # Logging options
 _LOGGING_SECTION = 'Logging'
 
```

Comment:
@LukeC92 @dkillick @pelson @pp-mo @djkirkham 
I know that this entire set of logging options is not deprecated, but while we are working towards Iris 2.0 and have also just implemented Python logging (with a view to further development), I really think we should take a long hard look at whether we still need this and should absolutely remove it if we don't.

Reply:
Yep. ``import_logger`` can be removed long-term.
Definitely a separate PR, but I'd probably support it happening for 2.0 without a deprecation warning. We will have to document the breaking change though (and think about the impact for people who have a logging section in their config - would it now break if we removed it, or just be ignored silently?).

All questions to be answered outside of this PR.